package java_0129;

class ExceptionEx2 {

	public static void main(String[] args) {
		
		try{
			int data = Integer.parseInt(args[0]);
			System.out.println(50/data);
		}catch(NumberFormatException ne){
			System.out.println("숫자가 아니다");
		}catch(ArrayIndexOutOfBoundsException ae){
			System.out.println("매개변수가 없다.");
		}catch(ArithmeticException ae2){
			System.out.println("0으로 나눴다.");
		}
	}

}
